src/main/resources/index.jelly

```diff
@@ -0,0 +1,10 @@
+<?jelly escape-by-default='true'?>
+<div>
+    <p>
+        Allows configuring users' permissions to projects organized in
+        folders from the Cloudbees Folder plugin. See the
+        <a href="https://github.com/jenkinsci/folder-auth-plugin">
+            GitHub page
```

Comment:
Is this better?

Reply:
Yes. GitHub link is probably not needed as well, Jenkins Update Center injects Wiki or GitHub link by default IIRC

Reply:
I verified it, the title of the plugin gets the link to the GitHub page. Thanks!